api/hydrophoneApi.go

```diff
@@ -126,6 +134,12 @@ func (a *Api) SetHandlers(prefix string, rtr *mux.Router) {
 	send.Handle("/signup/{userid}", varsHandler(a.sendSignUp)).Methods("POST")
 	send.Handle("/forgot/{useremail}", varsHandler(a.passwordReset)).Methods("POST")
 	send.Handle("/invite/{userid}", varsHandler(a.SendInvite)).Methods("POST")
+	// POST /confirm/send/team-invite
+	send.Handle("/team-invite", varsHandler(a.SendTeamInvite)).Methods("POST")
+	// PUT /confirm/send/team-invite - add admin role
+	send.Handle("/team-invite", varsHandler(a.UpdateTeamInvite)).Methods("PUT")
```

Comment:
This is not a modification of a team invite but a notification for a new role. This should be a different route.
```suggestion
	send.Handle("/membership-role", varsHandler(a.UpdateTeamInvite)).Methods("PUT")
```

Reply:
And I would used verb POST as this is the creation of a new notification